stix_shifter_modules/sumologic/tests/stix_translation/test_sumologic_stix_to_query.py

```diff
@@ -141,6 +141,14 @@ def test_user_account_created_query(self):
                   "\"toTime\": \"%s\"}" % (from_time, to_time)
         _test_query_assertions(query, queries)
 
+    def test_in_operator(self):
+        stix_pattern = "[user-account:display_name IN ('abc', 'def')]"
+        query = translation.translate('sumologic', 'query', '{}', stix_pattern)
+        _, from_time, to_time = query_constructor.convert_timestamp(query)
+        queries = "{\"query\": \"displayName = (abc OR def)\", \"fromTime\": \"%s\", \"toTime\": \"%s\"}" \
```

Comment:
are you sure this is a valid translated query? have you checked the API if it has any IN operator or similar kind of operator support?  if there's no IN operator support exists then it should work like OR operator.

Reply:
Hi Azam, 

I refer to the sumologic query doc here https://help.sumologic.com/05Search/Get-Started-with-Search/How-to-Build-a-Search/Keyword-Search-Expressions#examples

IN operator is NOT supported naturally from sumologic query, we will need to translate it to an combination of "=" and "OR", there are some examples in the docs using "=" and "OR"